server/server.go

```diff
@@ -248,6 +256,42 @@ func (server *HTTPServer) voteOnRule(writer http.ResponseWriter, request *http.R
 	}
 }
 
+func (server *HTTPServer) validateClusterAndRuleID(
+	clusterID types.ClusterName, ruleID types.RuleID, writer http.ResponseWriter,
+) bool {
+	clusterExists, err := server.Storage.CheckIfClusterExists(clusterID)
+	if err != nil {
+		log.Err(err)
+		responses.Send(http.StatusInternalServerError, writer, err.Error())
+		return false
+	}
+	if !clusterExists {
+		responses.Send(
+			http.StatusNotFound,
+			writer,
+			(&storage.ItemNotFoundError{ItemID: clusterID}).Error(),
+		)
+		return false
+	}
+
+	ruleExists, err := server.Storage.CheckIfRuleExists(ruleID)
+	if err != nil {
+		log.Err(err)
```

Comment:
Shouldn't it be `log.Error().Err(err)` ? Is it the same, just shorter?

Reply:
not sure

Reply:
fixed